src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #eef;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 0 17px;
+
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  padding: 32px 3px 0;
+  width: 160px;
+  height: 134px;
```

Comment:
you should not set width/heigh explicitly in such cases (correlates with the checklist, point 8)